### PR TITLE
Enabled kube-proxy on master node

### PR DESCRIPTION
**Pull request by piosz**
Partially addresses #9623. I tested it manually on GCE only.

cc @fgrzadkowski, @pwais, @kubernetes/goog-cluster

### CODE REVIEW

**Comment by thockin** [comment]
Does this work in the hosted-master case?

**Comment by roberthbailey** [comment]
> Does this work in the hosted-master case?

No.

File: cluster/saltbase/salt/top.sls [hunk 1]
```diff
@@ -72,6 +72,7 @@ base:
     - docker
     - kubelet
 {% endif %}
+    - kube-proxy
```
Comment (pwais) [inline cluster/saltbase/salt/top.sls]: Minor nit: would this be better on line 44 before all the optional configuration?  (In that case, perhaps lines 53-56 could move up too).  Not sure if I think about these configs like the original authors do, but my argument is that it's easier to discover the defaults if they're not intermixed with the optional features.

**Comment by piosz** [comment]
Since I'm no longer need it closing the PR. If there is any reason to enable kube-proxy on master node please re-open it.

**Pull request closed without merging**